src/browser/modules/D3Visualization/mapper.ts

```diff
@@ -25,7 +25,9 @@ import { optionalToString } from 'services/utils'
 
 const mapProperties = (_: any) => Object.assign({}, ...stringifyValues(_))
 const stringifyValues = (obj: any) =>
-  Object.keys(obj).map(k => ({ [k]: optionalToString(obj[k]) }))
+  Object.keys(obj).map(k => ({
+    [k]: obj[k] === null ? 'null' : optionalToString(obj[k])
```

Comment:
Might want to get rid of the `optionalToString` function as it's used only here. Not necessary though